js/bulkhead-callback.js

```diff
@@ -279,16 +283,73 @@ var bulkheadCallBack = (function() {
         return match;
     };
 
+    var __validateEditorContent_BulkheadStep = function(content) {
+        var match = false;
+        try {
+            var pattern = "@Bulkhead\\(50\\)\\s*public Service serviceForVFA";
+            var regExpToMatch = new RegExp(pattern, "g");
+            content.match(regExpToMatch)[0];
+            match = true;
+        } catch (ex) {
+
+        }
+        return match;
+    };
+
+    var __validateEditorContent_AsyncBulkheadStep = function(content) {
+        var match = false;
+        try {
+            var pattern1 = "@Asynchronous\\s*@Bulkhead\\(value\\s?=\\s?50,\\s*" + 
```

Comment:
why are we matching exactly one whitespace between the "value" and "=", and between "=" and "50?